src/components/rocket/Rocket.js

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import Badge from 'react-bootstrap/Badge';
+import RocketsBookingButton from './RocketsBookingButton';
+import './Rocket.css';
+
+const Rocket = ({ rocket }) => (
+  <div className="container d-flex">
+    <div>
+      <img className="rocketImages" src={rocket.flickr_images[0]} alt={rocket.name} />
+    </div>
+    <div className="container">
+      <h3>{rocket.rocket_name}</h3>
+      <p>
+        {
+        rocket.reserved
+          ? <Badge pill bg="info">Reserved</Badge>
+          : ''
+          }
```

Comment:
-  Please use React conditional rendering syntax for conditions instead of the Ternary operator as this is part of the requirements.
You can use the syntax:
`{ rocket.reserved && ( // code for reserved rocket)}
{ !rocket.reserved && ( // code for not reserved rocket)}`